cfgmgr/teammgr.cpp

```diff
@@ -541,6 +541,8 @@ task_process_status TeamMgr::addLag(const string &alias, int min_links, bool fal
         return task_need_retry;
     }
 
+    m_lagKeys[alias] = m_lacp_key++;
```

Comment:
I need to make lacp_key from portchannel name, otherwise after warm reboot we might got wrong lacp_key